src/elements/code-snippet.ts

```diff
@@ -48,15 +50,17 @@ export class VCode extends HTMLElement {
   }
 
   render = () => {
-    const language = this.getAttribute('language') ?? '';
+    language = this.getAttribute('language') ?? '';
```

Comment:
Why are we moving these vars to the outer scope? If there are multiple snippets on the same page we'll have one code snippet's `code` and `language` vars overriding another.